packages/core/contracts/oracle/implementation/Staker.sol

```diff
@@ -381,6 +331,19 @@ abstract contract Staker is StakerInterface, Ownable, Lockable, MultiCaller {
      *          INTERNAL FUNCTIONS          *
      ****************************************/
 
+    // This function must be called before any tokens are staked. Update the voter's pending stakes when necessary.
+    // The inheriting contract from Staker must implement this logic by overriding this function.
```

Comment:
```suggestion
    // The inheriting contract from VotingV2 must implement this logic by overriding this function.
```

Reply:
I meant that the contract that inherits from `Staker`, in this case `VotingV2`, must implement this function and overwrite it. Do you agree with this? Does it have the same meaning as the change you propose?

Reply:
prehaps then do this
```suggestion
    // The contract that inherits from Staker (eg the VotingV2 contract) must implement this logic by overriding this function.
```